From 71d1eefaab32dcd221caa562cfc0943a39b9f503 Mon Sep 17 00:00:00 2001
From: JD Lien <jd@jdlien.com>
Date: Mon, 19 Oct 2026 16:03:12 -0600
Subject: [PATCH 2/4] PREVIEW, not for ak820pro-jdlien: Lights out's slash in
 white ('/', bulb_cols' fifth colour), so it parts from the slate outline it
 crosses

---
 keyboards/a_jazz/ak820pro/graphics/display.c | 16 ++++++++--------
 1 file changed, 8 insertions(+), 8 deletions(-)

diff --git a/keyboards/a_jazz/ak820pro/graphics/display.c b/keyboards/a_jazz/ak820pro/graphics/display.c
index 762b2e7cdc..0b989e9d68 100644
--- a/keyboards/a_jazz/ak820pro/graphics/display.c
+++ b/keyboards/a_jazz/ak820pro/graphics/display.c
@@ -1639,10 +1639,10 @@ static const char bulb_art[POWER_MODE_COUNT][BULB_H][BULB_W + 1] = {
         "..#*******#..", "...#*****#...", "....#***#....", "....=====....",
         "....+++++....", ".....+++.....",
     },
-    [POWER_MODE_MINIMAL] = {   /* PREVIEW: unlit, with a slash through it */
-        ".............", ".............", ".##..........", "..##.###.....",
-        "...##...##...", "...###...#...", "..#..##...#..", "..#...##..#..",
-        "..#....##.#..", "...#....##...", "....#...###..", "....=====.##.",
+    [POWER_MODE_MINIMAL] = {   /* PREVIEW: unlit, with a white slash ('/') through it */
+        ".............", ".............", ".//..........", "..//.###.....",
+        "...//...##...", "...#//...#...", "..#..//...#..", "..#...//..#..",
+        "..#....//.#..", "...#....//...", "....#...#//..", "....=====.//.",
         "....+++++....", ".....+++.....",
     },
 };
@@ -1662,12 +1662,12 @@ static const char bulb_art[POWER_MODE_COUNT][BULB_H][BULB_W + 1] = {
 #define COL_BULB_OFF        0x8D18   /* #8EA2C6: Lights out's outline, slate */
 #define COL_BULB_OFF_BASE   0x6BD2   /* #6B7994: Lights out's base, a darker slate */
 #define COL_BULB_OFF_COLLAR 0x4AAD   /* #4C566A: Lights out's collar */
-/* Per mode: '#', '*', '+', '='. */
-static const uint16_t bulb_cols[POWER_MODE_COUNT][4] = {
+/* Per mode: '#', '*', '+', '=', '/'. */
+static const uint16_t bulb_cols[POWER_MODE_COUNT][5] = {
     [POWER_MODE_FULL]    = {COL_FG, 0, COL_BULB_BASE, COL_BULB_COLLAR},
     [POWER_MODE_NORMAL]  = {COL_FG, 0, COL_BULB_BASE, COL_BULB_COLLAR},
     [POWER_MODE_SAVER]   = {COL_FG, COL_BULB_DIM, COL_BULB_BASE, COL_BULB_COLLAR},
-    [POWER_MODE_MINIMAL] = {COL_BULB_OFF, 0, COL_BULB_OFF_BASE, COL_BULB_OFF_COLLAR},
+    [POWER_MODE_MINIMAL] = {COL_BULB_OFF, 0, COL_BULB_OFF_BASE, COL_BULB_OFF_COLLAR, COL_FG},
 };
 #define SLOT_NO_BULB 0xFFu
 
@@ -1717,7 +1717,7 @@ static void draw_bolt_slot(bool absent, bool chrg, uint8_t bulb) {
         for (uint8_t y = 0; y < BULB_H; y++)
             for (uint8_t x = 0; x < BULB_W; x++) {
                 char c = bulb_art[bulb][y][x];
-                if (c != '.') tile[y * ICON_SLOT_W + x] = bulb_cols[bulb][c == '#' ? 0 : c == '*' ? 1 : c == '+' ? 2 : 3];
+                if (c != '.') tile[y * ICON_SLOT_W + x] = bulb_cols[bulb][c == '#' ? 0 : c == '*' ? 1 : c == '+' ? 2 : c == '=' ? 3 : 4];
             }
     }
     /* CPU pushes share SPI0 with the DMA blits. lcd_blit_ram quiesces the bus
-- 
2.54.0

